copter/source/docs/traditional-helicopter-swashplate-setup.rst

```diff
@@ -159,8 +159,27 @@ angle measures the desired maximum collective blade pitch.
 Set :ref:`H_SV_MAN <H_SV_MAN>` to 4 to have swashplate move to minimum position.  Adjust :ref:`H_COL_MIN <H_COL_MIN>` until blade pitch
 angle measures the desired minimum collective blade pitch.
 
+Version 4.2
+-----------
+Minimum and Maximum Collective Blade Pitch Angle
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+For :ref:`H_COL_ANG_MAX <H_COL_ANG_MAX>`, enter the maximum collective blade pitch angle in degrees that corresponds to :ref:`H_COL_MAX <H_COL_MAX>`.  For :ref:`H_COL_ANG_MIN <H_COL_ANG_MIN>`, enter the minimum collective blade pitch angle in degrees that corresponds to :ref:`H_COL_MIN <H_COL_MIN>`.
+
+Zero Thrust Point
++++++++++++++++++
+
+:ref:`H_COL_ZERO_THRST <H_COL_ZERO_THRST>` is the collective blade pitch angle in degrees for zero thrust.  The collective blade pitch that corresponds to zero thrust is used to set the mixing of collective to yaw.  This will be zero degrees collective blade pitch angle for symmetric blades. Set the servo manual setting :ref:`H_SV_MAN <H_SV_MAN>` to 3 to have swashplate move to zero thrust position and verify the collective blade pitch corresponds the pitch angle set for :ref:`H_COL_ZERO_THRST <H_COL_ZERO_THRST>`.
+
+Minimum Landed Collective
++++++++++++++++++++++++++
+
+The :ref:`H_COL_LAND_MIN <H_COL_LAND_MIN>` parameter is used as the lower collective pitch limit in degrees for modes that use altitude hold in the vertical axis.  This keeps the autopilot from driving collective to low resulting in ground resonance. The default angle for :ref:`H_COL_LAND_MIN <H_COL_LAND_MIN>` parameter is set to -2 degrees collective blade pitch angle.  If you have non-symmetrical blades, then set it to a blade pitch that is lower by the amount used as the zero thrust pitch angle.  For example, if the :ref:`H_COL_ZERO_THRST <H_COL_ZERO_THRST>` is determined to be -3 deg then :ref:`H_COL_LAND_MIN <H_COL_LAND_MIN>` should be set to -5 degrees.
```

Comment:
```suggestion
The :ref:`H_COL_LAND_MIN <H_COL_LAND_MIN>` parameter is used as the lower collective pitch limit in degrees for modes that use altitude hold in the vertical axis.  This keeps the autopilot from driving collective too low resulting in ground resonance. The default angle for :ref:`H_COL_LAND_MIN <H_COL_LAND_MIN>` parameter is set to -2 degrees collective blade pitch angle, and would allow symmetrical blade equipped vehicles to descend at a reasonable rate, yet avoid being too negative on the ground .  If you have non-symmetrical blades, then set it to a blade pitch that is lower by the amount used as the zero thrust pitch angle.  For example, if the :ref:`H_COL_ZERO_THRST <H_COL_ZERO_THRST>` is determined to be -3 deg then :ref:`H_COL_LAND_MIN <H_COL_LAND_MIN>` should be set to ~ -5 degrees.
```

Reply:
> perhaps we should move the note on measuring with H_SV_MAN rather than just sitting in a stab mode up into the 4.2 section to be sure its not missed...

I will move it up above 4.2 for the setting of H_COL_MIN and H_COL_MAX

> also, I dont know if you have made the landing detection bullet proof yet such that getting a big updraft can force detect and that land_min plays a role it that detection? if it still does, perhaps a note on that?

I think being able to set the default at -2 deg without user interaction makes it much more robust.  I will draft a note